challenge.js

```diff
@@ -34,14 +33,14 @@
  *	Using the same HTML elements as above...
  *
  *	Suppose A is the body element, B, C, and D are all divs,
- *		and E is a p. 
+ *		and E is a p.
  *
  *	Write a CSS selector that can be used to retrieve E from the DOM.
  *
  *	Answer below.
- */
+ */A D E {}
```

Comment:
You're used to seeing CSS selectors in a CSS style sheet, where they're followed by curly braces containing the styling rules. Your answer here is using the hierarchy of the elements, which is good. However, it is not using the tag names you were given to use. Here's an explanation of how to approach this:

Using only tag names and parent/child relationships, here is a solution:

`body > div > p`

`>` is the immediate descendant operator in CSS. So we're looking for a `p` that is the child of a `div`, which is the child of the `body`.

Note that this won't necessarily select *only* E.

3/4